saleor/graphql/order/bulk_mutations/order_bulk_create.py

```diff
@@ -0,0 +1,1064 @@
+                shipping_price_net_amount = Decimal(shipping_price.net)
+                shipping_price_gross_amount = Decimal(shipping_price.gross)
+                shipping_price_tax_rate = (
+                    shipping_price_gross_amount / shipping_price_net_amount - 1
+                )
+            else:
+                lookup_key = f"shipping_price_{delivery_method.shipping_method.id}"
+                db_price_amount = object_storage.get(lookup_key) or (
+                    ShippingMethodChannelListing.objects.values_list(
+                        "price_amount", flat=True
+                    )
+                    .filter(
+                        shipping_method_id=delivery_method.shipping_method.id,
+                        channel_id=channel.id,
+                    )
+                    .first()
+                )
+                if db_price_amount:
+                    shipping_price_net_amount = Decimal(db_price_amount)
+                    shipping_price_gross_amount = Decimal(
+                        shipping_price_net_amount * (1 + shipping_price_tax_rate)
+                    )
+                    object_storage[lookup_key] = db_price_amount
+
+        # calculate lines
+        order_total_gross_amount = Decimal(
+            sum((line.total_price_gross_amount for line in order_lines))
+        )
+        order_undiscounted_total_gross_amount = Decimal(
+            sum((line.undiscounted_total_price_gross_amount for line in order_lines))
+        )
+        order_total_net_amount = Decimal(
+            sum((line.total_price_net_amount for line in order_lines))
+        )
+        order_undiscounted_total_net_amount = Decimal(
+            sum((line.undiscounted_total_price_net_amount for line in order_lines))
+        )
+
+        return OrderAmounts(
+            shipping_price_gross=shipping_price_gross_amount,
+            shipping_price_net=shipping_price_net_amount,
+            shipping_tax_rate=shipping_price_tax_rate,
+            total_gross=order_total_gross_amount,
+            total_net=order_total_net_amount,
+            undiscounted_total_gross=order_undiscounted_total_gross_amount,
+            undiscounted_total_net=order_undiscounted_total_net_amount,
+        )
+
+    @classmethod
+    def get_delivery_method(
+        cls, input: Dict[str, Any], errors: List[OrderBulkError], object_storage
+    ) -> Optional[DeliveryMethod]:
+        warehouse, shipping_method, shipping_tax_class = None, None, None
+        shipping_tax_class_metadata, shipping_tax_class_private_metadata = None, None
+
+        is_warehouse_delivery = input.get("warehouse_id")
+        is_shipping_delivery = input.get("shipping_method_id")
+
+        if is_warehouse_delivery and is_shipping_delivery:
+            errors.append(
+                OrderBulkError(
+                    message="Can't provide both warehouse and shipping method IDs.",
+                    field="deliveryMethod",
```

Comment:
I'm wondering if it shouldn't be in snake case. In most of ValidationError we use snake case for the error field and then it's changes into camel case